packages/node-cli/src/commands/manageaccount/ManageAccountInputHandler.js

```diff
@@ -48,13 +48,21 @@ const ANSWERS_NAMES = {
 
 module.exports = class ManageAccountInputHandler extends BaseInputHandler {
 	constructor(options) {
-        super(options);
-        this._sdkPath = options.sdkPath;
+		super(options);
+		this._sdkPath = options.sdkPath;
 	}
 
 	async getParameters(params) {
+		if (!this._runInInteractiveMode) {
+			return params;
+		}
+		let answers;
 		const authIDActionResult = await getAuthIds(this._sdkPath);
-		let answers = await this._selectAuthID(authIDActionResult.data, prompt);
+		if (params.hasOwnProperty(COMMAND.OPTIONS.INFO)) {
```

Comment:
When is this case happening it we have stop the non-interactive cases before?

Reply:
ups... I was doing some tryouts and forget to change again... I fix it now